app/controllers/application_controller.rb

```diff
@@ -2,4 +2,8 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def not_found
+    raise ActionController::RoutingError.new('Not Found')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.